sdk/core/src/main/java/com/klaviyo/core/networking/KlaviyoNetworkMonitor.kt

```diff
@@ -23,7 +24,9 @@ internal object KlaviyoNetworkMonitor : NetworkMonitor {
     /**
      * List of registered network change observers
      */
-    private var networkChangeObservers = mutableListOf<NetworkObserver>()
```

Comment:
One more unbelievably nitpicky detail, but lets just keep the name `networkChangeObservers` without `Concurrent` at the end, like you did with the other 4.